src/tests/Microsoft.PowerFx.Interpreter.Tests/Scenarios/ScenarioDotNetObjectWrapper.cs

```diff
@@ -48,7 +48,7 @@ private static void Add(RecalcEngine engine, string name, object obj)
         [InlineData("Index(array, 0)", "#error")] // Out of bounds, low
         [InlineData("Index(array, -1)", "#error")] // Out of bounds, low
         [InlineData("Index(array, 100)", "#error")] // Out of bounds, high
-        [InlineData("Text(obj.Value)", "#error")] // cast error. 
+        [InlineData("Text(obj.Value)", "10")] // cast error. 
```

Comment:
>cast error [](http://example.com/codeflow?start=49&length=10)

Not an error anymore, can you update the comment?

Reply:
Done